test/extensions/transport_sockets/alts/tsi_socket_test.cc

```diff
@@ -14,9 +14,11 @@ namespace TransportSockets {
 namespace Alts {
 namespace {
 
+using testing::InSequence;
 using testing::NiceMock;
 using testing::Return;
 using testing::ReturnRef;
+using testing::StrictMock;
```

Comment:
From clang tidy: Message: using decl 'StrictMock' is unused

Reply:
Done.